File: tkpdseller/src/main/java/com/tokopedia/seller/product/edit/data/source/cloud/model/editproductform/Info.java

package com.tokopedia.seller.product.edit.data.source.cloud.model.editproductform;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Info {

    @SerializedName("product_returnable")
    @Expose
    private int productReturnable;
    @SerializedName("shop_has_terms")
    @Expose
    private int shopHasTerms;

    public int getProductReturnable() {
        return productReturnable;
    }

    public void setProductReturnable(int productReturnable) {
        this.productReturnable = productReturnable;
    }

    public int getShopHasTerms() {
        return shopHasTerms;
    }

    public void setShopHasTerms(int shopHasTerms) {
        this.shopHasTerms = shopHasTerms;
    }

}

File: features/home/account-home/src/main/java/com/tokopedia/home/account/analytics/data/model/Info.kt
package com.tokopedia.home.account.analytics.data.model


import com.google.gson.annotations.SerializedName

data class Info(
    @SerializedName("shop_id")
    val shopId: String = "",
    @SerializedName("shop_name")
    val shopName: String = ""
)